docs/udp.rst

```diff
@@ -433,3 +433,131 @@ which we can process further and download::
     cube.download("fancy.tiff", format="GTiff")
 
 See :ref:`datacube_from_process` for more information on :func:`~openeo.rest.connection.Connection.datacube_from_process`.
+
+
+.. _workflow_example:
+
+Working Example: Create EVI Timeseries UDP
+==========================================
+
+In this first example, we'll create an openEO workflow script,
+including declaring parameters, building, storing and loading a UDP
+
+Here, let us consider that we want to recalculate the EVI-timeseries
+as done in :ref:`example-use-case-evi-map-and-timeseries` but for a different area.
+Therefore for future, instead of redefining the whole process, we simply 
+want to define it as a UDP ``EVI_timeseries``, that can be easily shared 
+and reused later without dealing with the entire workflow each tim:
+
+.. code-block:: python
+
+    import openeo
+    from openeo.api.process import Parameter
+
+    # Create connection to openEO back-end
+    connection = openeo.connect("...").authenticate_oidc()
+
+    # declaring parameters
+    temporal_interval = Parameter(
+        name="temporal_interval",
+        description="The date range to load.",
+        schema={"type": "array", "subtype": "temporal-interval"},
+        default =["2018-06-15", "2018-06-27"]
+    )
+    bbox = Parameter(
+        name="bbox",
+        description="The bounding box to load.",
+        schema={"type": "object", "subtype": "geojson"},
```

Comment:
`"subtype": "geojson"` is for GeoJSON objects, while this is a west-south-east-north bbox, which is not GeoJSON

Reply:
Also: If you use `aggregate_spatial`  already, it is not necessary to use `spatial_extent` in `load_collection` because that will be inferred automatically.
Dropping `bbox`  will make this example more compact and to the point I think

Reply:
Noted. Removed the parameter  bbox within the load collection and used feature alone